### PR TITLE
Dev install doc

### PR DESCRIPTION
**Pull request by jphnoel**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -14,20 +14,27 @@ uData customizations for Etalab / Data.gouv.fr.
...
+Install [udata](https://github.com/opendatateam/udata).
 
-.. code-block:: console
+Remain in the same virtual environment (for Python) and use the same version of npm (for JS).
 
-    $ pip install udata-gouvfr
+Install **udata-gouvfr**:
 
-or with easy_install:
+    cd
```
Comment (davidbgk) [inline README.rst]: I think that line is unnecessary given that you probably don't want to install the project within your home folder (do you?)

File: README.rst [hunk 2]
```diff
@@ -14,20 +14,27 @@ uData customizations for Etalab / Data.gouv.fr.
...
+    git clone ​https://github.com/etalab/udata-gouvfr.git
+    pip install -e udata-gouvfr
 
-.. code-block:: console
+Modify the local configuration file of **udata** (typically, *~/udata/udata.cfg*) as following:
```
Comment (davidbgk) [inline README.rst]: Which leads to that path, I would remove the `~/` part.

**Comment by davidbgk** [comment]
Note that Travis doesn't build because of a non-ascii character within the README, I can't see it though so it's probably an unbreakable space or something like that. Happy bug hunting :)

**Comment by jphnoel** [comment]
I did the following in a Python shell:

    fd = open('README.rst)
    x = fd.read()
    x
    >>>  "... git clone \xe2\x80\x8bhttps://github.com/etalab/udata-gouvfr.git..."

If I check in Emacs (**M-x describe-char**) what the hell is going on between **clone** and **https**, I got the following:

    preferred charset: unicode (Unicode (ISO10646))
    code point in charset: 0x200B
    syntax: which means: whitespace
    file code: #xE2 #x80 #x8B (encoded by coding system utf-8-unix)
    Character code properties: customize what to show
    name: ZERO WIDTH SPACE

Only God knows how I inserted this odd character..

**Comment by jphnoel** [comment]
Can someone invite me in the etalab team, so that I can merge the PR ?

**Comment by davidbgk** [comment]
@jphnoel actually, your invitation is pending. Go to https://github.com/etalab/ to accept it :)

**Pull request merged**